### DISCUSSION TITLE
How to add LLDP ZTP VLAN at specific L3Leaf Port via AVD

### DISCUSSION THREAD

**Discussion by geckigek** [aristanetworks/avd#3997] [Q&A] [answered]
Hi there,

I want to use the ztp feature to tell a 710P, which has no mgmt interface, to contact CVP to get a inital config to be deployed via AVD. There is a guide how to enable ztp via lldp on the l3leaf switch, which is the uplink switch for the 710P L2 leaf.(https://www.arista.com/en/support/toi/eos-4-27-1f/14879-campus-ztp-through-vlan-advertised-from-spine-aggregation-switch)

Is there a way to define via AVD on the downlink port of the l3leaf the lldp ztp vlan for the layer2 switch?

Best regards

Stefan

**Comment by ClausHolbechArista** [comment, marked as answer]
Since L2 switches in AVD are always using port-channels, we have chosen to implement inband ZTP support using LACP fallback individual.
Set `inband_ztp: true` under `l2leaf` either in defaults, node-group or node settings for the 710P (notice it is for the switch to be booted via inband_ztp, not the uplink switch).
Also make sure you have configured inband management with `inband_mgmt_subnet` or `inband_mgmt_ip` and notice that the default `inband_mgmt_vlan` is 4092. I suggest you read the (long) descriptions around the inband_mgmt_* parameters, since there are a number of restrictions and useful explanations.

Let us know if it works out or if you have more questions.

Thanks
Claus

**Reply by ClausHolbechArista** [reply]
@jrecchia1029 it would be nice if you could find time to write a guide for inband ZTP under the `eos_designs` how-to guides.

**Comment by geckigek** [comment]
Hi,

thanks for your answer. So with the port-channel configuration it is not possible to use the lldp ztp vlan feature directly on the port? Correct? And how do I verify that die lacp fallback individual is configured correctly? Which commands show be found in the rendered cli config for the l3leaf and l2leaf switch?

Best regards

**Reply by ClausHolbechArista** [reply]
The ethernet interfaces part of a downlink port-channel will have the configuration on them that is effective in the fallback case. The port-channel will have the fallback individual command. The downstream switch (the one booting with inband ztp) will not have any special config since it has no config during inband ztp :)
